packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -147,6 +147,13 @@ const getUsersElement = (
 ): JSX.Element => {
   const userElements: JSX.Element[] = [];
   typingUsers.forEach((user, index) => {
+    if (user.displayName) {
+      let truncatedDisplayName = user.displayName;
+      if (truncatedDisplayName.length > 50) {
+        truncatedDisplayName = truncatedDisplayName.substring(0, 50) + '...';
+      }
+      user.displayName = truncatedDisplayName;
+    }
```

Comment:
Maybe we can simply the code as below:
`  typingUsers.forEach((user, index) => {
    let truncatedDisplayName = user.displayName;

    if (truncatedDisplayName && truncatedDisplayName.length > 50) {
      truncatedDisplayName = truncatedDisplayName.substring(0, 50) + '...';
    }
    userElements.push(
      onRenderUser ? (
        onRenderUser(user)
      ) : (
        <Text className={mergeStyles(userDisplayNameStyles)} key={`user-${index}`}>
          {truncatedDisplayName}
        </Text>
      )
    );
    userElements.push(<Text key={`comma-${index}`}>{`${delimiter}`}</Text>);
  });`

Reply:
Good suggestion!